examples/misc_exporter_gltf.html

```diff
@@ -35,6 +35,7 @@
 			<label><input id="option_drawrange" name="visible" type="checkbox" checked="checked"/>Truncate drawRange</label>
 			<label><input id="option_binary" name="visible" type="checkbox">Binary (<code>.glb</code>)</label>
 			<label><input id="option_forceindices" name="visible" type="checkbox">Force indices</label>
+			<label><input id="option_forcepot" name="visible" type="checkbox">Force PTO textures</label>
```

Comment:
`Force PTO` ☺

Reply:
@mrdoob Done! Force PTO is what I need :_D